ionic_user_interface/src/components/gym-selector/GymSelector.vue

```diff
@@ -72,19 +87,23 @@ import {
   IonSelectOption,
   IonRow,
   IonCol,
-  IonButton,
 } from '@ionic/vue';
+import { Geolocation } from '@capacitor/geolocation';
 import Lookup, { Country } from 'country-code-lookup';
-import { map, mapOutline, warning } from 'ionicons/icons';
+import { map, mapOutline, warning, locateOutline } from 'ionicons/icons';
+import haversine from 'haversine';
 
 import MessageBox from '@/components/MessageBox.vue';
+import LoadingButton from '@/components/LoadingButton.vue';
 import GymMap from '@/components/GymMap.vue';
-import getGymsByCountry, { GymLocation } from '@/common/api/route/getGymsByCountry';
+import getGymsByCountry, { LatLong, GymLocation } from '@/common/api/route/getGymsByCountry';
+import getReverseGeoLocationCached from '@/common/api/map/getReverseGeolocation';
 import AutoComplete from './AutoComplete.vue';
```

Comment:
`@/components/gym-selector/AutoComplete.vue`

Reply:
These components are part of the same larger component, so I don't think you should use the absolute path